api/api_test.go

```diff
@@ -25,6 +23,18 @@ func TestSetup(t *testing.T) {
 	})
 }
 
+func TestClose(t *testing.T) {
+	Convey("Given an API instance", t, func() {
+		ctx := context.Background()
+		api := Setup(context.Background(), mux.NewRouter(), &mongo.JobStore{})
```

Comment:
Again, use a jobstorer mock instead of the mongo package.

Reply:
thanks for pointing this out - done